### PR TITLE
Task/fep 686 2

### PR DESCRIPTION
**Pull request by tcrite1**
- Sets output directory
- Removes tree shaking check (for now)
- Ensures types get bundled

### CODE REVIEW

**Review by cafloyd — APPROVED** [review]

File: docs/SelectMulti.stories.ts [hunk 1]
```diff
@@ -11,11 +11,11 @@ export default {
 const Template = (args, { argTypes }) => ({
 	props: Object.keys(argTypes),
 	components: { SelectMulti },
-	data() { return { values: [] } },
+	data() { return { selected: [] } },
 	template: `
 	<div class="wrapper">
 		<SelectMulti
-			v-model:values="values"
+			v-model:values="selected"
```
Comment (cafloyd) [inline docs/SelectMulti.stories.ts:18]: I think this is what you were saying to me on Slack, but just want to be doubly sure - it's ok that `values` and `selected` don't match here (and in SelectSingle.stories.ts), right? It doesn't have to be `v-model:selected="selected"`?
I tried running your branch locally and it looks fine but just checking!

**Comment by wickkidd** [comment]
I recommend using interactive rebase to `fixup` a lot of those commits so that the history would reflect what you figured out rather than how you did so. I can walk you though it if you're unfamiliar.

File: docs/SelectMulti.stories.ts (hunk 1 above)
Comment (tcrite1) [reply docs/SelectMulti.stories.ts:18]: @cafloyd that's correct.

**Pull request merged**